themes/inbeat/layouts/partials/site/mixin/unlockCalculator.html

```diff
@@ -0,0 +1,73 @@
+<script>
+  var unlockCalculator = {
+    data() {
+      return {
+        showPopup: false,
+        unlocked: false,
+        verifying: false,
+        popupClass: "",
+        submissions: 0,
+      };
+    },
+    computed: {
+      cookieName() {
+        return "unlock-tool" + window.location.pathname.replace(/\//g, "-");
+      },
+    },
+    methods: {
+      showUnlockPopup() {
+        let cookieValue = getCookie(this.cookieName);
+
+        if (!cookieValue) {
+          setCookie(this.cookieName, 1, 60);
+          return false;
+        } else if (cookieValue === "1") {
+          this.showPopup = true;
+          this.popupClass = "show";
+          this.handleState(true);
+          return true;
+        } else {
+          return false;
+        }
+      },
+      hideUnlockPopup() {
+        setCookie(this.cookieName, true, 60);
+        this.showPopup = false;
+        this.handleState(false);
+      },
+      handleState(newVal) {
+        if (newVal) {
+          this.addBlur();
+          this.preventScroll();
+        } else {
+          this.removeBlur();
+          this.allowScroll();
+        }
+      },
+      startVerifying() {
+        this.verifying = true;
+        this.unlocked = true;
+        setTimeout(() => {
+          this.verifying = false;
+        }, 10000);
+      },
+      addBlur() {
+        document.querySelectorAll("section").forEach(function (item) {
+          item.classList.add("blur");
+        });
```

Comment:
Every `section` on the site?

Reply:
Yes, we want to blur out the page when popup show - But actually, I recheck the design and we don't blur so I'll remove it